### DISCUSSION TITLE
Support postgresql

### DISCUSSION THREAD

**Discussion by yezz123** [fastapi/fastapi#7125] [Questions]
```python 
from couchbase import LOCKMODE_WAIT
from couchbase.bucket import Bucket
from fastapi import FastAPI
```
I can use this lib when i'm working with `PostgreSQL`?

**Comment by luebke-dev** [comment]
couchbase is a nosql database and postgres is sql.

if you want to use postgres use an orm like sqlalchemy or https://github.com/encode/orm if you need async support

**Comment by tiangolo** [comment]
Thanks for the help @SebastianLuebke! And thanks for closing the issue @yezz123! 🍰 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
